src/components/MainLayout.tsx

```diff
@@ -1,14 +1,15 @@
+import Box from '@mui/material/Box';
 import HeaderContainer from '../ui/common/Header/HeaderContainer';
 import MainContainer from '../ui/common/Main/MainContainer';
 import SidebarContainer from '../ui/common/Sidebar/SidebarContainer';
 
 const MainLayout = () => {
   return (
-    <>
+    <Box>
```

Comment:
I think Box is extra you can use <></> syntax

Reply:
Yes, <Box> syntax has been replaced